sources/platform/monitoring/index.md

```diff
@@ -1,94 +1,108 @@
+3. **Value** - The value of the metric violating the condition and triggering the alert.
+4. **Run ID** - The ID of the run that triggered the alert which links directly to the run detail in Apify Console.
+5. **Actor** - The full name of the actor that triggered the alert which links to the actor detail in Apify Console.
+6. **Task** - If the monitoring alert was set up for a task, then this field will contain the name of the task which links to the task detail in Apify Console.
 
-* [Monitoring shared datasets](./monitor_shared_datasets.md)
+![Email notification](./images/email-notification.png)
 
-* [Monitoring multiple actors or tasks](./monitor_multiple_tasks.md)
+While the in app notification will contain less information, it will point you directly to the actor or task that triggered the alert:
 
-## How do I turn it off or make changes? {#how-do-i-turn-it-off-or-make-changes}
+ <img src={require("./images/in-app-notification.png").default} title="In app notification" width="70%" />
 
-To turn off a monitoring task, simply run it with the **Mode** (in the task's **Input** section) set to **Delete configuration**. It will clean up all its resources and turn off all monitoring activity. You will no longer receive [notifications](#notifications).
+## Other
 
-To make any changes to an existing configuration, choose the **Update configuration** mode in the task's **Input**, make your changes and run the monitoring suite task again.
+### What should I monitor when scraping?
 
-### Add the monitoring suite to your account {#add-the-monitoring-suite-to-your-account}
+There are many metrics you might want to monitor when you are scraping the web. Here are some examples:
 
-On the monitoring suite's [page](https://apify.com/apify/monitoring) in [Apify Store](https://apify.com/store), click the **Try for free** button. This will take you to [Apify Console](https://console.apify.com). Add the suite to your [list of Actors](https://console.apify.com/actors) and create a new [task](../actors/running/tasks.md).
+**Data quality**:
 
-![Monitoring actor in Apify Store](./images/monitoring-in-store.png)
+1. **Number of results** returned by your solution. This is useful if you are scraping a list of products, for example. You can set up an alert to notify you if the number of results is lower than expected. Which indicates that something changed on the website you are scraping.
+2. **Number of fields** returned. This is something that indicates a change of the website. For example the manufacturer name moved to other place.
 
-## Notifications {#notifications}
+**Performance**:
 
-The suite will send notifications to the [email address associated with your account](https://console.apify.com/account) by default.
+1. **Duration** of the run. If your solution is taking longer than usual to finish, you can set up an alert to notify you, this will help you prevent your solution from being stuck and from wasting resources.
```

Comment:
```suggestion
1. **Duration** of the run. If your solution is taking longer than usual to finish, you can set up an alert to notify you. This will help you prevent your solution from being stuck and from wasting resources.
```